cmd/mesh/manifest-versions.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 const (
-	versionsMapURL = "https://raw.githubusercontent.com/istio/operator/master/version/versions.yaml"
+	versionsMapURL = "https://raw.githubusercontent.com/morvencao/operator/br_update_versions_map/version/versions.yaml"
```

Comment:
Could you please mark this with DONOTMERGE or something so it doesn't sneak it?

Reply:
since #305 is merged, @morvencao can we get this updated so this PR can be merged?